src/MySql/Clause/RowListClause.php

```diff
@@ -7,6 +7,9 @@
 use AlephTools\SqlBuilder\MySql\Expression\RowListExpression;
 use AlephTools\SqlBuilder\Sql\Clause\ValuesClause;
 
+/**
+ * @property RowListExpression|null $values
+ */
 trait RowListClause
 {
     use ValuesClause;
```

Comment:
> @AlephTav
Here, I have intentionally omitted the type to be able to more flexibly replace the expression type in clauses.

In case of expression subtypes it can be replaced by `@property` annotation like this.